### PR TITLE
(DOCSP-24698) Atlas CLI release notes for 1.1.6 and 1.1.7

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=62fe7852b354ebcf055ca9f4)
[Jira Ticket 1](https://jira.mongodb.org/browse/DOCSP-24679)
[Jira Ticket 2](https://jira.mongodb.org/browse/DOCSP-24698)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-24698/atlas-cli-changelog/)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,31 @@
...
+
+{+atlas-cli+} 1.1.7
+-------------------
+
+Released 18 August 2022
+
+- Adds support for the following platforms:
+
+  - RHEL 9
+  - Ubuntu 22.04 (Jammy)
```
Comment (JuliaMongo) [inline source/atlas-cli-changelog.txt:25]: I've never seen the version names mentioned before, so not sure about Jammy.

**Review by JuliaMongo — APPROVED** [review]
LGTM, and thank you, Sarah! Left one question.

File: source/atlas-cli-changelog.txt (hunk 1 above)
Comment (sarahsimpers) [reply source/atlas-cli-changelog.txt:25]: We include the version names on the Apt tab of the [install page](https://www.mongodb.com/docs/atlas/cli/upcoming/install-atlas-cli/), but I can take it out, the version # will suffice. Thanks!

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**